extension/persistence/relational-jdbc/src/main/resources/db.properties

```diff
@@ -0,0 +1,31 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+jdbc.url=jdbc:postgresql://localhost:5432/postgres
+jdbc.username=testuser
+jdbc.password=password
+jdbc.driverClassName=org.postgresql.Driver
+dbcp2.initialSize=5
+dbcp2.maxTotal=20
+dbcp2.maxIdle=10
+dbcp2.minIdle=2
+dbcp2.maxWaitMillis=5000
```

Comment:
Are these settings configurable outside of this file?... e.g. at Polaris start time?

Reply:
sure, i think will need to come-up with a mechanism like this 

```  
-Dpolaris.persistence.eclipselink.configuration-file=/path/to/persistence.xml \
```

like in eclipse link so that once can override the properties from Polaris in start time.

Reply:
Quarkus already provides an extensive configuration framework. Let's leverage that.